tests/unit/terra.test.js

```diff
@@ -0,0 +1,188 @@
+import { Client } from '@liquality/client'
+import { ChainNetworks } from '@/utils/networks'
+import { TerraNetworks } from '@liquality/terra-networks'
+import { TerraRpcProvider } from '@liquality/terra-rpc-provider'
+import { TerraWalletProvider } from '@liquality/terra-wallet-provider'
+import { TerraSwapProvider } from '@liquality/terra-swap-provider'
+import { TerraSwapFindProvider } from '@liquality/terra-swap-find-provider'
+
+test('Terra Network TesnetConfig', () => {
```

Comment:
Ugh, this feels like a very weird test 😅 
Validating the content of a completely static object

Reply:
done